Description: Misc patches to build process to pickup
 unversioned jar files to support use of system libraries
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: lucene-solr/lucene/contrib/benchmark/build.xml
===================================================================
--- lucene-solr.orig/lucene/contrib/benchmark/build.xml	2012-05-23 11:15:26.156364644 +0100
+++ lucene-solr/lucene/contrib/benchmark/build.xml	2012-05-23 11:15:29.248374135 +0100
@@ -131,11 +131,11 @@
       <pathelement path="${highlighter.jar}"/>
       <pathelement path="${analyzers-common.jar}"/>
       <pathelement path="${facet.jar}"/>
-      <fileset dir="${common.dir}/contrib/icu/lib" includes="icu4j-4.8.1.1.jar"/>
+      <fileset dir="${common.dir}/contrib/icu/lib" includes="icu4j.jar"/>
       <path refid="base.classpath"/>
     	<fileset dir="lib">
-    	  <include name="commons-compress-1.2.jar"/>
-    	  <include name="xercesImpl-2.9.1.jar"/>
+    	  <include name="commons-compress.jar"/>
+    	  <include name="xercesImpl.jar"/>
     	</fileset>
     </path>
     <path id="run.classpath">
@@ -176,7 +176,7 @@
 	<path id="collation.runtime.classpath">
 	  <path refid="run.classpath"/>
     <pathelement path="${icu.jar}"/>
-    <fileset dir="${common.dir}/contrib/icu/lib" includes="icu4j-4.8.1.1.jar"/>
+    <fileset dir="${common.dir}/contrib/icu/lib" includes="icu4j.jar"/>
 	</path>
 	
 	<target name="collation" depends="compile,jar-icu,top-100k-wiki-word-files">
Index: lucene-solr/lucene/contrib/icu/build.xml
===================================================================
--- lucene-solr.orig/lucene/contrib/icu/build.xml	2012-05-23 11:15:26.156364644 +0100
+++ lucene-solr/lucene/contrib/icu/build.xml	2012-05-23 11:15:29.248374135 +0100
@@ -28,7 +28,7 @@
   <import file="../contrib-build.xml"/>
 
   <path id="icujar">
-     <pathelement location="lib/icu4j-4.8.1.1.jar"/>
+     <pathelement location="lib/icu4j.jar"/>
   </path>
 
   <path id="classpath">
Index: lucene-solr/lucene/contrib/analyzers/phonetic/build.xml
===================================================================
--- lucene-solr.orig/lucene/contrib/analyzers/phonetic/build.xml	2012-05-23 11:15:26.156364644 +0100
+++ lucene-solr/lucene/contrib/analyzers/phonetic/build.xml	2012-05-23 11:15:29.248374135 +0100
@@ -30,7 +30,7 @@
   <import file="../../contrib-build.xml"/>
 
   <path id="classpath">
-    <pathelement path="lib/commons-codec-1.6.jar"/>
+    <pathelement path="lib/commons-codec.jar"/>
     <path refid="base.classpath"/>
   </path>
 </project>
Index: lucene-solr/lucene/common-build.xml
===================================================================
--- lucene-solr.orig/lucene/common-build.xml	2012-05-23 11:15:29.228374072 +0100
+++ lucene-solr/lucene/common-build.xml	2012-05-23 11:37:06.720300803 +0100
@@ -59,14 +59,14 @@
   <property name="ivy.resource" value="org/apache/ivy/ant/antlib.xml" />
   <available resource="${ivy.resource}" property="ivy.available" />
 
-  <property name="junit.jar" value="junit-4.10.jar"/>
+  <property name="junit.jar" value="junit4.jar"/>
   <property name="junit-location.jar" value="${common.dir}/test-framework/lib/${junit.jar}"/>
   <path id="junit-path">
     <pathelement location="${junit-location.jar}"/>
   </path>
 
   <path id="ant-path">
-    <fileset dir="${common.dir}/test-framework/lib" includes="ant-*.jar"/>
+    <fileset dir="${common.dir}/test-framework/lib" includes="ant*.jar"/>
   </path>
 
   <!-- default arguments to pass to JVM executing tests -->
Index: lucene-solr/lucene/contrib/analyzers/kuromoji/build.xml
===================================================================
--- lucene-solr.orig/lucene/contrib/analyzers/kuromoji/build.xml	2012-05-23 11:15:26.156364644 +0100
+++ lucene-solr/lucene/contrib/analyzers/kuromoji/build.xml	2012-05-23 11:15:29.256374158 +0100
@@ -64,7 +64,7 @@
   </target>
 
   <path id="tools.dependencies">
-    <fileset dir="../../icu/lib" includes="icu4j-4.8.1.1.jar"/>
+    <fileset dir="../../icu/lib" includes="icu4j.jar"/>
   </path>
 
   <path id="tools.classpath">
Index: lucene-solr/lucene/contrib/queries/build.xml
===================================================================
--- lucene-solr.orig/lucene/contrib/queries/build.xml	2012-05-23 09:50:45.088081982 +0100
+++ lucene-solr/lucene/contrib/queries/build.xml	2012-05-23 11:37:22.776348534 +0100
@@ -26,7 +26,7 @@
   <import file="../contrib-build.xml"/>
 
   <path id="classpath">
-    <pathelement path="lib/jakarta-regexp-1.4.jar"/>
+    <pathelement path="lib/jakarta-regexp.jar"/>
     <path refid="base.classpath"/>
   </path>
 
